packages/gatsby-plugin-theme-ui/README.md

```diff
@@ -18,10 +18,11 @@ prevent a flash of unstyled colors when using color modes.
 
 ## Options
 
-| Key           | Default value | Description                                                                                                                                                                                                                                        |
-| ------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `prismPreset` | `null`        | The name of the preset you'd like to use to style code blocks inside your markdown files. The available presets can be found in the [theme-ui docs](https://theme-ui.com/packages/prism/). You can also use a package string of your own choosing. |
-| `preset`      | `null`        | This can be a JSON theme object or a string package name. Make sure the package you're requiring is installed in your dependencies.                                                                                                                |
+| Key             | Default value | Description                                                                                                                                                                                                                                        |
+| --------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `prismPreset`   | `null`        | The name of the preset you'd like to use to style code blocks inside your markdown files. The available presets can be found in the [theme-ui docs](https://theme-ui.com/packages/prism/). You can also use a package string of your own choosing. |
+| `preset`        | `null`        | This can be a JSON theme object or a string package name. Make sure the package you're requiring is installed in your dependencies.                                                                                                                |
+| `noFlashScript` | `true`        | Setting this option to `false` will stop inserting the `theme-ui-no-flash` script in your pages. Useful for AMP (Accelerated Mobile Pages) sites.                                                                                                  |
```

Comment:
```suggestion
| `noFlashScript` | `true`        | By default, the plugin injects a `script` tag to prevent color mode flashing. Set this option to `false` to omit the script. Useful for AMP (Accelerated Mobile Pages) pages. |
```

Reply:
Thanks, renamed it